pwndbg/wrappers/__init__.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import pwndbg.which
+
+
+class OnlyWithCommand(object):
+    def __init__(self, func):
+        self.func = func
```

Comment:
This is missing `functools.wraps` or something similar (don't remember if that's the proper way to wrap using a class decorator) and so makes wrapped function not having proper metadata.

E.g. it discards `func.__name__`:
```python
In [3]: class Decorator(object):
   ...:     def __init__(self, *a): pass
   ...:     
   ...: 

In [4]: @Decorator
   ...: def foo():
   ...:     pass
   ...: 

In [5]: foo.__name__
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-5-5f8bc38be3ae> in <module>()
----> 1 foo.__name__

AttributeError: 'Decorator' object has no attribute '__name__'

In [6]: def bar(): pass

In [7]: bar.__name__
Out[7]: 'bar'
```